packages/purpose-platformstate-writer/src/utils.ts

```diff
@@ -0,0 +1,128 @@
+import {
+  DeleteItemCommand,
+  DeleteItemInput,
+  DynamoDBClient,
+  GetItemCommand,
+  GetItemCommandOutput,
+  GetItemInput,
+  PutItemCommand,
+  PutItemInput,
+} from "@aws-sdk/client-dynamodb";
+import { unmarshall } from "@aws-sdk/util-dynamodb";
+import {
+  genericInternalError,
+  PlatformStatesCatalogEntry,
+  PlatformStatesEServiceDescriptorPK,
+  PlatformStatesPurposeEntry,
+  PlatformStatesPurposePK,
+} from "pagopa-interop-models";
+import { config } from "./config/config.js";
+
+export const writePlatformPurposeEntry = async (
```

Comment:
Since there write, read and delete are almost identical to the ones [here](https://github.com/pagopa/interop-be-monorepo/blob/main/packages/catalog-platformstate-writer/src/utils.ts#L32) would it be beneficial to declare these as generics and move to `commons`?

Reply:
Moving to commons is being planned, but I wouldn't turn those into functions with generics, because it doesn't seem a straightforward operation, considering the differences between those entries.